cv/pothole_detection/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+cmake_minimum_required(VERSION 3.0.2)
+project(pothole_detection)
+
+# # Find catkin macros and libraries
+find_package(catkin REQUIRED COMPONENTS 
+  rospy 
+  std_msgs 
+  geometry_msgs
+  )
+
```

Comment:
Also on a side note, for the `FloatList` msg, we need to generate the msg so it's viewable in C++

Do we want to do this in here or in VL PR ? @1999michael for vis